sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientTests.cs

```diff
@@ -61,6 +73,53 @@ public void DeleteIssuerArgumentValidation()
             Assert.AreEqual("issuerName", ex.ParamName);
         }
 
+        [Test]
+        public void SetContactsArgumentValidation()
+        {
+            ArgumentException ex = Assert.ThrowsAsync<ArgumentNullException>(() => Client.SetContactsAsync(null));
+            Assert.AreEqual("contacts", ex.ParamName);
+
+            // Below test failed since SetContactsAsync() method doesn't assert empty.
+            // Consider assert contacts empty scenario ?
+            List<CertificateContact> contacts = new List<CertificateContact>();
+            ex = Assert.ThrowsAsync<ArgumentNullException>(() => Client.SetContactsAsync(contacts));
```

Comment:
Test failed, should this assert contacts empty ?

Reply:
Null != empty. At best, this might throw an `ArgumentException` if we do validate empty, but you should test whatever the docs say we do. I'm good about documenting what exceptions are thrown and why.